mxcube3/__init__.py

```diff
@@ -23,134 +23,82 @@
 from flask_socketio import SocketIO
 from flask_session import Session
 
-# To make "from mxcubecore import ..." possible
-fname = os.path.dirname(__file__)
-sys.path.insert(0, fname)
-
-from mxcubecore import HardwareRepository as hwr
-
-hwr.add_hardware_objects_dirs([os.path.join(fname, "HardwareObjects")])
-
-import app as mxcube
-from config import Config
+from mxcube3.config import Config
+from mxcube3.app import MXCUBEApplication
```

Comment:
Introduced a MXCUBEApplication object that encapsulates (or represents) "application" wide data and functions

Reply:
its a neat what of separating domains. I like it